### DISCUSSION TITLE
Working with the Build Tree

### DISCUSSION THREAD

**Discussion by rumplestilzken** [rumplestilzken/gargoyle_lineageos20#62] [Announcements]
This discussion is for concerns with working with the android build tree.

**Comment by CliffracerX** [comment]
The big two from my studies so far:

- What ``lunch`` parameters are required for unlisted Unihertz devices?  (E.G, the Jelly Star)
- How do you fill out the necessary ``vendor/`` files for a Unihertz device when building, if you do so at all?
